includes/class-wc-payments-features.php

```diff
@@ -78,6 +78,9 @@ public static function is_upe_split_enabled() {
 	 */
 	public static function is_upe_deferred_intent_enabled() {
 		$account = WC_Payments::get_database_cache()->get( WCPay\Database_Cache::ACCOUNT_KEY, true );
+		if ( null === $account ) {
+			return true;
+		}
 		return is_array( $account ) && ( $account[ self::DEFERRED_UPE_SERVER_FLAG_NAME ] ?? false );
```

Comment:
Side question - would it also make sense to return `true` in case `$account[ self::DEFERRED_UPE_SERVER_FLAG_NAME ]` is not set?

i.e.:
```suggestion
		return is_array( $account ) && ( $account[ self::DEFERRED_UPE_SERVER_FLAG_NAME ] ?? true );
```

Asking because we're basically saying: "if there's no account data - force dUPE". But if `$account` data is an array without the `is_deferred_intent_creation_upe_enabled` key, we're saying that dUPE is disabled. Am I getting my neurons too twisted? 😵

Reply:
You're right and https://github.com/Automattic/woocommerce-payments/pull/7686/commits/abc7347b03a246c0caf99daf5973562cc5f4bea7 improves this together with removing one redundant test which came up after the change, thank you!

The scenario with returning disabled dUPE state wouldn't happen because [server always returns a value](https://github.com/Automattic/woocommerce-payments-server/blob/trunk/server/wp-content/rest-api-plugins/endpoints/wcpay/service/class-account-service.php#L1976-L1978), but this doesn't mean client can rely on this so I agree with you.